parsl/executors/taskvine/executor.py

```diff
@@ -363,6 +366,7 @@ def submit(self, func, resource_specification, *args, **kwargs):
 
         # Also consider any *arg that looks like a file as an input:
         input_files += [self._register_file(f) for f in args if isinstance(f, File)]
+        logger.debug(f"registered input files {input_files}")
```

Comment:
for debug logs in Parsl, @khk-globus and I have been pushing on a style that defers string interpolation until the log system has decided if it really wants to render that message - often that's not the case and rendering can be expensive. that format looks like this:

https://github.com/Parsl/parsl/blob/64e163ceaf4b43746909f30c9659738f29dd84e1/parsl/channels/local/local.py#L58

rather than using Python-level string formatting (either f-strings or `"".format(`